_posts/2019-04-25-mar-updates.md

```diff
@@ -0,0 +1,42 @@
+---
+title: March Updates -  A new Kubernetes operator & more Cloud Native Apps. 
+author: Hong Phuc Dang 
+date: 2019-04-25
+banner:
+  image: mar-banner.jpg
+---
+### This is a recap of open source activities and development at Zalando in the month of March
+
+## Project Highlights 
+
+A new operator is added to Zalando’s list of Cloud Native Applications.  [Es-Operator](https://github.com/zalando-incubator/es-operator) - an operator for running Elasticsearch in Kubernetes with focus on operational aspects, like safe draining and offering auto-scaling capabilities for Elasticsearch data nodes, rather than just abstracting manifest definitions.
+
+To make things even simpler for developers, we also released a new framework that helps to build Kubernetes operators in Python. [Kopf - Kubernetes Operator Pythonic Framework](https://github.com/zalando-incubator/kopf) - a  framework and a library to make Kubernetes operators development easier, just in few lines of Python code. The main goal is to bring the Domain-Driven Design to the infrastructure level, with Kubernetes being an orchestrator/database of the domain objects (custom resources), and the operators containing the domain logic (with no or minimal infrastructure logic).
+
+**Weekly Open Source Maintainer Day at Developer Productivity** - Pionioneer in open source development at Zalando, Developer Productivity has actively released free software and contributed to many upstream projects. Since the beginning of this year, the Cloud Infrastructure team leading by [Jannis Rake-Revelant](https://twitter.com/jannis_r), reserved every Tuesday for open source maintaining work - which they called ‘Maintainer Tuesday’. It is a part of Zalando commitment to open source community. 
```

Comment:
this is internal - we have channels for this, so we should avoid mixing what is useful for internal and external audiences

Reply:
I think it is relevant to external. Google broadcast to the world that they give engineers 20% to work on their own ideas. It is to promote the working environment at Zalando.

Reply:
I agree, it's worth a small mention that a team dedicates one full day to OSS.

Reply:
Should it be "pioneer" or is "pionioneer" intentional?
```suggestion
**Weekly Open Source Maintainer Day at Developer Productivity** - Pioneer in open source development at Zalando, Developer Productivity has actively released free software and contributed to many upstream projects. Since the beginning of this year, the Cloud Infrastructure team leading by [Jannis Rake-Revelant](https://twitter.com/jannis_r), reserved every Tuesday for open source maintaining work - which they called ‘Maintainer Tuesday’. It is a part of Zalando commitment to open source community. 
```

Reply:
Fair arguments, then re-write with an external audience in mind:

1. Explain what "developer productivity" means, ex. projects they maintain etc. so there is context for people not aware of internal structures
2. provide more information - how many developers are doing this every week, maybe there are already tangible examples of how this benefits external users of our projects (ex: due to this we able to release X and invest Y number of hours into maturing Z) 

Especially when we use claims such as "commitment to open source" and "pioneering" we should back up those claims with tangible examples - which I know we can for sure.

Reply:
Thank you for the feedback. I'll see if I can add any additional info. However this is a blog article  - not a scientific paper - I think it is also fine if we don't have evidence for every sentence :)

Reply:
No need for scientific level info... - just provide a context so external readers understand what this means